### ISSUE TITLE
Can scribd-downloader create epubs?

### ISSUE THREAD

**Issue by hub2git** [ritiek/scribd-downloader#46]
I'm a premium account user of Scribd. Is there a way to get scribd-downloader to create ePub files (ebook format) from the books on Scribd?

Thank you.

**Comment by ritiek** [comment]
Nope. There are however other tools available for converting markdown or pdf to epub.

**Comment by hub2git** [comment]
Ah, so that's what ".md" means. Glad that there are other tools available.

Now I just have to the mitmproxy working.... I'm having problems with that program. Can you recommend other/newbie-friendly programs? I run Linux Mint 19.1 Tessa.

**Comment by ritiek** [comment]
There are other tools I've heard similar to mitmproxy like charlesproxy. I can't say if they are easier to use as I've only worked with mitmproxy. A quick web search for "similar tools like mitmproxy" should give you a list of them.

I'll have some free time this week, so I'll probably make a short video/screencast and improve the README section on using premium headers and cookies. Maybe I'll also be able to implement #40 during this time.

**Comment by hub2git** [comment]
Yes, number #40 would be amazing! Thank you. :100:

**Comment by WangLeeds** [comment]
I think we can make the feature of creating ePub files into enhancement, Ritiek.

**Comment by hub2git** [comment]
Thank you so much. I looked hard to find this project, and it's getting better than ever.

**Comment by ritiek** [comment]
@hub2git @WangLeeds Are any of you guys willing to share a Scribd premium account (email to me) so I could work on this? The guy who shared his credentials with me in #38 changed his password (or maybe the account expired?), so I can no longer use that account to test stuff out.

If you can't for some reason, no worries at all.

**Comment by WangLeeds** [comment]
I have sent you already, Ritiek.

**Comment by hub2git** [comment]
Hi @ritiek, I see that  @WangLeeds has beat me to the punch. :)  I also can share my account if ever you still need it. :+1:

**Comment by ritiek** [comment]
> I think we can make the feature of creating ePub files into enhancement, Ritiek.

To be honest, this is no easy task. We're lucky to have [md2pdf](https://github.com/jmaupetit/md2pdf) for creating PDF files but there doesn't seem a developed python library for converting stuff to epub. Also, I'd like to keep this tool as a bare minimum way to download stuff from Scribd.

For now I'd recommend using stable external tools like [pandoc](https://pandoc.org/) or online services for converting markdown or pdf to epub format.

**Comment by hub2git** [comment]
Oh, no problem. :smile: 

I'll take a look at pandoc next week... or whenever #40 is implemented.

Have you been able to access premium content, thanks to @WangLeeds's help?

**Comment by ritiek** [comment]
> Have you been able to access premium content, thanks to @WangLeeds's help?

Yep, thank you @WangLeeds!
